docs/content/integrations/dbt/reference.mdx

```diff
@@ -154,6 +154,8 @@ models:
 
 Then the asset key for a model named `some_model` will be `marketing/some_model`. If you haven't configured a custom schema, then the asset key will be `some_model`.
 
+Note that Dagster allows the optional specification of a [`code_version`](/guides/dagster/scheduling-assets#step-5-change-code-versions) for each software-defined asset, which is used to track asset staleness. The `code_version` for an asset arising from a dbt model is defined automatically as the hash of the SQL defining the DBT model. This means that changing the code of the model will automatically cause the corresponding asset, and all downstream assets, to be marked stale.
```

Comment:
Can this go under a heading, something like "dbt models, code versions, and staleness"? That will make it easier to link to.